protocol/bin/flow.js

```diff
@@ -92,6 +138,9 @@ function capitalize (s) {
 }
 
 function analyzeMessages (json, project) {
+  // ui means incoming
+  const isUIProtocol = ['notifyCtl'].indexOf(json.protocol) === -1 && !!json.protocol.match(/^notify.*|.*ui$/i)
```

Comment:
we keep track of whats a ui call or not. this means we can filter out the outgoing calls (the ...rpc()) and the incomingCallMap into the 2 types.